test/clj/cider/nrepl/middleware/debug_test.clj

```diff
@@ -70,16 +70,6 @@
 
 (deftest read-debug-command
   (reset! @#'d/debugger-message {})
-  ;; Check that :next is the first
-  (with-redefs [t/send (fn [trans {:keys [key input-type]}]
-                         (deliver (@d/promises key) (first input-type)))]
-    (is (= 'value (#'d/read-debug-command 'value {}))))
-
-  ;; Check that :quit is the last
-  (binding [*msg* {:session (atom {})}]
-    (with-redefs [d/abort! (constantly :aborted)
-                  d/read-debug (fn [r c & _] (last c))]
-      (is (= :aborted (#'d/read-debug-command 'value {})))))
```

Comment:
What order are we displaying these options now? I'm fine with deleting  the tests, as long as the frontend does something to maintain the old order.